Competition2017/src/competition/subsystems/shooter/SideShooterSubsystem.java

```diff
@@ -20,14 +18,19 @@
     
     private static Logger log = Logger.getLogger(ShooterSubsystem.class);
     
-    private final XCANTalon masterMotor;
-    
+
+    protected final XCANTalon masterMotor;
+    public enum ShooterSide {
```

Comment:
there's now a RobotSide enum that we can use for all of the sided things like this we should use instead